editor.planx.uk/src/@planx/components/PlanningConstraints/List.tsx

```diff
@@ -107,71 +116,85 @@ function ConstraintListItem({ children, ...props }: any) {
             justifyContent: "space-between",
           }}
         >
-          {children}
           <Button
             disableRipple
             onClick={() =>
               setShowConstraintData((showConstraintData) => !showConstraintData)
             }
+            style={{
+              display: "flex",
+              flexDirection: "row",
+              justifyContent: "space-between",
+              width: "100%",
+              boxShadow: "none",
+              color: "#0B0C0C",
```

Comment:
Do we actually want to access the team color or a shared theme color here? Not sure I've seen this one come up before!